tests/common/test_event.py

```diff
@@ -213,31 +213,39 @@ def test_periodic_forwards_args(self, mock_event):
 
     def test_save_event(self):
         add_event('test', message='test event')
-        self.assertTrue(len(os.listdir(self.tmp_dir)) == 2)
+        self.assertTrue(len(os.listdir(self.tmp_dir)) == 1) # We now only send 1 event for event_id = 1
```

Comment:
minor: I'd remove or clarify this comment... "we now only..." - why? "event_id = 1" - what is id 1?